webui/app/css/base.css

```diff
@@ -372,4 +372,12 @@ pre {
 .metadata-modal-body {
     height: 600px;
     overflow-y: auto;
-}
\ No newline at end of file
+}
```

Comment:
these can be removed

Reply:
I guess you've commented the wrong line here. Did you mean the classes that make the DropZone invisible/visible? They were below, I removed them